docs/user-manual/paging.adoc

```diff
@@ -135,10 +141,10 @@ The configured policy will start based on the first value to reach its mark.
 
 ==== Maximum read from page
 
-`max-read-page-messages` and `max-read-page-bytes` are used to control messaging reading from paged file into the Queue.
-The broker will add messages on the Queue until either `max-read-page-meessages` or `max-read-page-bytes` reaches the limit.
+`max-read-page-messages`, `max-read-page-bytes`, `prefetch-page-messages` and `prefetch-page-bytes` are used to control reading from paged file into the Queue.
+The broker will add messages as long as all these limits are satisfied.
 
-If both values are set to -1 the broker will keep reading messages as long as the consumer is reaching for more messages.
+If all these values are set to -1 the broker will keep reading messages as long as the consumer is reaching for more messages.
```

Comment:
Its still just if `max-read-page-messages` and `max-read-page-bytes`are set to <=0, as it was before before, so its probably clearer to just say that.
```suggestion
If `max-read-page-messages` and `max-read-page-bytes` are set to -1 the broker will keep reading messages as long as the consumer is reaching for more messages. The `prefetch-page-messages` and `prefetch-page-bytes` will not be considered.
```

Reply:
I will actually update the code to use all these values.